### PR TITLE
feat: display transaction messages in airdrop steps

### PR DESCRIPTION
**Pull request by findolor**

### CODE REVIEW

File: contracts/cw20/merkleAirdrop/contract.ts [hunk 1]
```diff
@@ -316,5 +398,99 @@ export const CW20MerkleAirdrop = (
...
+
+    return {
+      instantiate,
+      registerAndReleaseEscrow,
+      depositEscrow,
+      claim,
+    }
+  }
+
+  return { instantiate, use, messages }
```
Comment (RyanWarner) [inline contracts/cw20/merkleAirdrop/contract.ts:495]: This file is huge. I like to keep files under 200 lines. Is it worth the time to break it up a bit?

File: pages/airdrops/register.tsx [hunk 2]
```diff
@@ -216,6 +230,17 @@ const RegisterAirdropPage: NextPage = () => {
           />
         )}
 
+        <Conditional
+          test={!!(airdrop && !airdrop.escrow && !airdrop.processing)}
```
Comment (RyanWarner) [inline pages/airdrops/register.tsx:234]: Minor nitpick but I find it more readable to break this out into a variable.

```
const showTransactionMessage = !!(airdrop && !airdrop.escrow && !airdrop.processing)
...
<Conditional
  test={showTransactionMessage}
```

**Comment by grikomsn** [comment]
This PR has many merge conflicts. I advise cherry pick commits one by one or remake from latest development branch.

File: contracts/cw20/merkleAirdrop/contract.ts (hunk 1 above)
Comment (findolor) [reply contracts/cw20/merkleAirdrop/contract.ts:495]: For now I think we can leave it like this but maybe come back in the future to split it

**Comment by findolor** [comment]
implemented in https://github.com/CosmosContracts/juno-tools/pull/123

**Pull request closed without merging**